multivac/gather_data.py

```diff
@@ -249,10 +249,10 @@ def get_compiler_version(log_file):
         return compiler
 
     @staticmethod
-    def get_time_in_queue(time_queued, time_started):
-        unix_time_queued = github_time_to_unix(time_queued)
+    def calc_time_diff(time_started: str, time_ended: str) -> float:
         unix_time_started = github_time_to_unix(time_started)
-        time_in_queue = unix_time_started - unix_time_queued
+        unix_time_ended = github_time_to_unix(time_ended)
+        time_in_queue = unix_time_ended - unix_time_started
```

Comment:
The name of this variable is strange. It is rather `diff` than `time_in_queue`.

Reply:
fixed